### PR TITLE
cmctl: 1.11.2 -> 1.13.0

### PR DESCRIPTION
**Pull request by wegank**
## Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [x] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/networking/cluster/cmctl/default.nix [hunk 1]
```diff
@@ -1,39 +1,52 @@
...
     "-X github.com/cert-manager/cert-manager/pkg/util.AppGitCommit=${src.rev}"
   ];
 
-  nativeBuildInputs = [ installShellFiles ];
+  nativeBuildInputs = [
+    installShellFiles
+  ];
+
+  # Trusted by this computer: no: x509: “cert-manager” certificate is not trusted
+  doCheck = false;
```
Comment (JoshVanL) [inline pkgs/applications/networking/cluster/cmctl/default.nix:38]: Could you explain why this is needed? Why will the tests not work?

**Review by JoshVanL — CHANGES_REQUESTED** [review]
Tanks for this @wegank 

I guess the update script no longer works/is needed as each cert-manager binary has its own go module now. Just a question from me.

File: pkgs/applications/networking/cluster/cmctl/default.nix (hunk 1 above)
Comment (JoshVanL) [reply pkgs/applications/networking/cluster/cmctl/default.nix:38]: FWIW, it built fine on my machine removing these lines (and running the checks).

Comment (wegank) [reply pkgs/applications/networking/cluster/cmctl/default.nix]: Indeed, the tests only fail on darwin. Adding `cacert` or enabling `__darwinAllowLocalNetworking = true;` doesn't help either, so I give up.

**Review by JoshVanL — APPROVED** [review]

**Pull request merged**